### ISSUE TITLE
Default value for Boolean mapping

### ISSUE THREAD

**Issue by quangnpd** [duydb/deneric#3]
I have an entity mapping Boolean property with default value is "true"
```class Outline extends deneric.Entity {
constructor (params = {}) {
    super(params, {
...
      visible: ['visible', deneric.Boolean, true],
...
    })
  }
}
```
And when I create an instance like:
```
let outline = new Outline({
visible: false
})
```
the `outline` entity always has `visible` property is `true`

**Comment by duydb** [comment]
Hi @quangnpd!
I have been fix this issue. Please check the test case. Thanh you

**Comment by duydb** [comment]
> I have an entity mapping Boolean property with default value is "true"
> 
> ```
> constructor (params = {}) {
>     super(params, {
> ...
>       visible: ['visible', deneric.Boolean, true],
> ...
>     })
>   }
> }
> ```
> And when I create an instance like:
> 
> ```
> let outline = new Outline({
> visible: false
> })
> ```
> the `outline` entity always has `visible` property is `true`

This

> Hi @quangnpd!
> I have been fix this issue. Please check the test case. Thanh you

This issue fix by version [1.0.2-beta](https://www.npmjs.com/package/deneric/v/1.0.2-beta)
